### PR TITLE
Revert "[MIRROR] Removes force from slappers"

### PR DESCRIPTION
**Pull request by Spacedong**
its 1 force you fuck it should be able to ddo at least 1 damage, plus, no force means it gets rid of the hitsound

### CODE REVIEW

**Comment by tortellinitony** [comment]
No. The sound is hardcoded, and the force was taken away so that you can't break stuff like windows with your bare hands.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -591,7 +591,7 @@
 	desc = "This is how real men fight."
 	icon_state = "latexballon"
 	item_state = "nothing"
-	force = 0
```
Comment (GuyonBroadway) [inline code/game/objects/items/weaponry.dm:594]: NOT MODULAR!!!! YOU KNOW HOW TO MAKE THIS MODULAR GODDAMNIT!!!

**Comment by deathride58** [comment]
Upstream butchered slappers pretty hard. Will be working on a PR that makes slappers great again soon, the bug with being able to break things via slaps will be fixed. Have patience, you hooligans.

**Comment by Chnkr** [comment]
@Spacedong I'll be closing this PR, you should *never* revert a mirror PR unless there is a serious reason to do so, these reasons do not include "hurr durr i dislike this feature"

I'd suggest you open a new PR that does this the proper way, which is to make it modular trough the `hippiestation` directory.

**Pull request closed without merging**